.changeset/smooth-worms-retire.md

```diff
@@ -0,0 +1,24 @@
+---
+"@clerk/clerk-js": minor
+"@clerk/clerk-react": minor
+"@clerk/types": minor
+---
+
+Extended the `UserButton` component to support custom menu items:
```

Comment:
Introduce support for custom menu items in `<UserButton/>`.

Reply:
Done